new-integration-tests/src/test/resources/wdt-models/model2-wls.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+domainInfo:
+    AdminUserName: '@@SECRET:__weblogic-credentials__:username@@'
+    AdminPassword: '@@SECRET:__weblogic-credentials__:password@@'
+    ServerStartMode: 'prod'
+
+topology:
+    Name: "wls-domain1"
```

Comment:
what is the difference between domainUid and this Name here ?

Reply:
I am not sure. I copied it from model1-wls.yaml file.

Reply:
The domain name is set in Domain() here:
Domain domain = new Domain()
        .apiVersion(API_VERSION)
        .kind("Domain")
        .metadata(new V1ObjectMeta()
            .name(domainUid)

It seems the topology name in the model file does not matter.